diff --git a/vendor/github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver/internal/host/hostinfo.go b/vendor/github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver/internal/host/hostinfo.go
index c93bf0312..144c91575 100644
--- a/vendor/github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver/internal/host/hostinfo.go
+++ b/vendor/github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver/internal/host/hostinfo.go
@@ -78,6 +78,7 @@ func NewInfo(containerOrchestrator string, refreshInterval time.Duration, logger
 	if err != nil {
 		return nil, fmt.Errorf("failed to create aws session: %w", err)
 	}
+	mInfo.awsConfig = cfg
 
 	mInfo.ec2Metadata = mInfo.ec2MetadataCreator(ctx, cfg, refreshInterval, mInfo.instanceIDReadyC, mInfo.instanceIPReadyC, logger)
 
